tests/affiliatedcertification/tests/affiliated_certification_operator.go

```diff
@@ -35,13 +34,100 @@ var _ = Describe("Affiliated-certification operator certification,", func() {
 		Expect(err).ToNot(HaveOccurred(), "Error creating namespace")
 
 		By("Deploy OperatorGroup")
-		err = affiliatedcerthelper.DeployOperatorGroup(affiliatedcertparameters.TestCertificationNameSpace,
-			utils.DefineOperatorGroup("affiliatedcert-test-operator-group",
+		if affiliatedcerthelper.IsOperatorGroupInstalled(affiliatedcertparameters.OperatorGroupName,
```

Comment:
In this "If" condition IsOperatorGroupInstalled func is used however in the next couple of the same conditions another function is consumed IsOperatorInstalled.

Reply:
This section deals with the operator group object, while the next deals with the operators themselves.